manifests/pipecd/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "pipecd.fullname" . }}-server
```

Comment:
The "server" suffix must be removed as defined in the helper

```suggestion
  name: {{ include "pipecd.fullname" . }}
```

Reply:
sorry, i fixed it.